### PR TITLE
[fix] More checkstyle error fixes.

### PR DESCRIPTION
**Pull request by Yamaguti**
Please merge [PR 35](https://github.com/rulojuka/sbking/pull/35) before reviewing this one.

After this change, we only will have:
1. 25 variable naming errors (all variables named `logger` in the codebase);
2. 25 Visibility modifiers on class variables; and
3. 157 magic numbers (which I think should be demoted to warnings)

### CODE REVIEW

**Review by rulojuka — COMMENTED** [review]
Are you sure about this spotbugs change?

File: pom-client.xml [hunk 1]
```diff
@@ -103,18 +103,35 @@
...
+					</execution>
+				</executions> -->
+			</plugin>
+
 		</plugins>
 	</build>
 
 	<reporting>
 		<plugins>
-			<!-- Bug Finding with SpotBugs -->
```
Comment (rulojuka) [inline pom-client.xml:111]: Are you sure about this spotbugs change?

File: sbking_checkstyle.xml [hunk 2]
```diff
@@ -161,16 +161,25 @@
 
     <!-- Checks for class design                         -->
     <!-- See https://checkstyle.org/config_design.html -->
-    <module name="DesignForExtension"/>
+    <module name="DesignForExtension">
+      <property name="severity" value="warning"/>
```
Comment (rulojuka) [inline sbking_checkstyle.xml:165]: Probably info is better. I really don't care much about that specific check.. Anyway, your call

**Review by rulojuka — APPROVED** [review]
Looks very good to me. About the remaining errors:
- There should be a better way to have only one logger in the application *while* knowing (and printing) the class it was called on the log. I will explore this for 30min before just renaming all of them to LOGGER.
- Visibility modifiers: these should be fixed too, I don't even know why I did that in the first place...
- Magic Numbers: I really like the idea of letting them as errors, but I could do that in the future (after fixing all of them). For now, warnings are good enough.
Please contact me on Discord telling me which one you will continue working so I can work on the other ones! Soon after each one is solved, we may uncomment the `pom.xml` to fail on errors! :)

**Comment by Yamaguti** [comment]
> Are you sure about this spotbugs change?

I reinstated the spotbugs on both pom files. I thought mvn site wans't working because of a misconfiguration on the <report> tag, but that appears to not be the issue.

> P.S.: Oops, I thought this would go on a specific commit..

I don't like commenting on commits because the push -f after a rebase make the comment dangling :)

File: pom-client.xml (hunk 1 above)
Comment (Yamaguti) [reply pom-client.xml]: As per commented on the PR, I reinstated both spotbugs configuration on the report tags

**Pull request merged**